### PR TITLE
use near-abi-rs

### PR DESCRIPTION
**Pull request by miraclx**
https://github.com/near/near-abi-rs/pull/1
https://github.com/near/near-sdk-rs/pull/889

### CODE REVIEW

**Review by austinabell — APPROVED** [review]

**Comment by miraclx** [comment]
One of our CI runs seems to have gotten stuck following an issue with GitHub Actions.

https://www.githubstatus.com/incidents/pmw828sz2w7k

https://github.com/orgs/community/discussions/30079#discussioncomment-3472190

File: cargo-near/Cargo.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ schemars = "0.8"
 
 [dependencies.near-abi]
 git = "https://github.com/near/near-abi-rs"
-rev = "4844ef4ff3e5cf3c3a8aa6565c3195a2339a345f"
+rev = "983346e8ce2a8304645a92ba7735f9a6eacceda7"
 features = ["__chunked-entries"]
```
Comment (austinabell) [inline cargo-near/Cargo.toml]: Can we use `0.1.0-pre.0`? https://crates.io/crates/near-abi/0.1.0-pre.0

Comment (miraclx) [reply cargo-near/Cargo.toml]: has already been updated.

Comment (austinabell) [reply cargo-near/Cargo.toml]: oh, oops, must have been looking at an outdated commit

**Review by austinabell — APPROVED** [review]

**Review by itegulov — APPROVED** [review]

File: integration-tests/templates/_Cargo.toml [hunk 2]
```diff
@@ -7,10 +7,14 @@ edition = "2021"
...
 [dependencies]
-near-sdk = { version = "4.1.0-pre.1", features = ["abi"] }
 serde = { version = "1", features = ["derive"] }
 schemars = "0.8"
 
+[dependencies.near-sdk]
+git = "https://github.com/near/near-sdk-rs.git"
+rev = "13fbfc9930fb0ee162dd9c726ce02e15219c956d"
+features = ["abi"]
+
```
Comment (miraclx) [inline integration-tests/templates/_Cargo.toml:17]: should probably wait for an sdk prerelease before merging this in, or wdyt?

Comment (itegulov) [reply integration-tests/templates/_Cargo.toml:17]: What is missing for us to release 4.1.0-pre.2? Are we waiting on https://github.com/near/near-sdk-rs/pull/891 and https://github.com/near/near-sdk-rs/pull/893 to be pulled in?

Comment (miraclx) [reply integration-tests/templates/_Cargo.toml:17]: We can cut the prerelease without those two. Since we still need to properly review the ABI embedding.

Comment (itegulov) [reply integration-tests/templates/_Cargo.toml:17]: Ok, I suggest we ask @austinabell to cut 4.1.0-pre.2 and wait for the release then

Comment (austinabell) [reply integration-tests/templates/_Cargo.toml:17]: ah, I'll do that now

Comment (austinabell) [reply integration-tests/templates/_Cargo.toml:17]: done

**Review by austinabell — APPROVED** [review]
updated versions lgtm

**Pull request merged**